wagtail/admin/static_src/wagtailadmin/scss/userbar.scss

```
@@ -111,6 +111,21 @@ $positions: (
     margin: 0;
   }
 
+  .w-userbar-axe-count {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-button-no;
+    border-radius: 100px;
+    color: $color-white;
+    font-size: 0.8125rem;
+    height: 20px;
+    width: 20px;
```

```suggestion
    height: theme('fontSize.5');
    width: theme('fontSize.5');
```

We also have a scale for sizing of elements. It’s not as useful, but at least has the benefit of using `rem` units for sizing, which is important here so the circle gets bigger when users increase their browser font size.